guacamole/src/main/webapp/translations/en.json

```diff
@@ -447,6 +447,7 @@
         "FIELD_OPTION_SECURITY_RDP"   : "RDP encryption",
         "FIELD_OPTION_SECURITY_TLS"   : "TLS encryption",
 
+        "FIELD_OPTION_SERVER_LAYOUT_DE_CH_QWERTZ" : "Swiss German (Qwertz)",
```

Comment:
What's the process for this ending up in the other translations? Is it up to the translators to determine that a new option has been added?

Reply:
Yeah, at this point, unless someone else can translate it to the other languages, it will only show up in the English translation file, and Guacamole will pull from that by default if it doesn't exist elsewhere.